lib/tasks/setup_application.rake

```diff
@@ -1,208 +0,0 @@
-require 'rake'
```

Comment:
Was this task deleted by accident? I don't think it should be a job since it's meant to be run immediately and only once, so we should leave it as-is.